rpki-validator/pom.xml

```diff
@@ -28,7 +28,7 @@
     <properties>
         <maven.build.timestamp.format>yyyy.MM.dd'.'HH.mm.ss</maven.build.timestamp.format>
         <xodus.version>1.3.124</xodus.version>
-        <jetty.version>9.4.15.v20190215</jetty.version>
+	<jetty.version>9.4.21.v20190926</jetty.version>
```

Comment:
Not aligned… But nice to find the latest working version

Reply:
Aligned.